modules/nw-monitoring-dpu-status.adoc

```diff
@@ -0,0 +1,71 @@
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-dpu-monitoring-status_{context}"]
+= Monitoring the status of DPU
+
+The `oc get dpu` command allows the user to see the current state of the DPU infrastructure. Follow this procedure to monitor the status of various cards.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* An account with `cluster-admin` privileges is available.
+* The DPU Operator is installed.
+
+.Procedure
+
+. Run the following command to check the overall health of your nodes:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+The example output provides a list of all nodes in the cluster along with their status. Ensure that all nodes are in the `Ready` state before proceeding.
++
+[source,terminal]
+----
+NAME                           STATUS   ROLES    AGE    VERSION
+ocpcluster-master-1            Ready    master   10d    v1.32.9
+ocpcluster-master-2            Ready    master   10d    v1.32.9
+ocpcluster-master-3            Ready    master   10d    v1.32.9
+ocpcluster-dpu-ipu-219         Ready    worker   42h    v1.32.9
+ocpcluster-dpu-marvell-41      Ready    worker   3d23h  v1.32.9
+ocpcluster-dpu-ptl-243         Ready    worker   3d23h  v1.32.9
+worker-host-ipu-219            Ready    worker   3d19h  v1.32.9
+worker-host-marvell-41         Ready    worker   4d     v1.32.9
+worker-host-ptl-243            Ready    worker   3d23h  v1.32.9
+----
++
+This output shows three master nodes, three worker nodes identified as for example `worker-host-ipu-219` and inside each of these worker nodes a DPU identified as for example `ocpcluster-dpu-ipu-219`.
+
+. Run the following command to report on the status of the DPUs:
++
+[source,terminal]
+----
+$ oc get dpu
+----
++
+The example output provides a list of detected DPUs.
++
+[source,terminal]
+----
+NAME                                 DPU PRODUCT                    DPU SIDE        MODE NAME               STATUS
+030001163eec00ff-host                Intel Netsec Accelerator       false           worker-host-ptl-243     True
+d4-e5-c9-00-ec-3v-dpu                Intel Netsec Accelerator       true            worker-dpu-ptl-243      True
+intel-ipu-0000-06-00.0-host          Intel IPU E2100                false           worker-host-ipu-219     False
+intel-ipu-dpu                        Intel IPU E2100                true            worker-dpu-ipu-219      False
+marvell-dpu-0000-87-00.0-host        Marvell DPU                    false           worker-host-marvell-41  True
+marvell-dpu-ipu                      Marvell DPU                    true            worker-dpu-marvell-41   True
+----
+* `DPU PRODUCT`: Displays the vendor or type of DPU for example Intel or Marvell.
+* `DPU SIDE`: Indicates whether the DPU is operating on the host side `false` or the DPU side `true`. Each physical DPU is represented twice.
+* `MODE NAME`: The name of the node where the DPU is located. The host worker node for `false` entries, the DPU node for `true` entries.
```

Comment:
The name of the node where the DPU is located. This is the host worker node for `false` entries and the DPU node for `true` entries.